### PR TITLE
don't limit blockvolume size to default blockhosting vol size

### PR DESCRIPTION
**Pull request by raghavendra-talur**
Signed-off-by: Raghavendra Talur <rtalur@redhat.com>

### CODE REVIEW

File: apps/glusterfs/app_block_volume_test.go [hunk 1]
```diff
@@ -178,11 +179,28 @@ func TestBlockVolumeLargerThanBlockHostingVolume(t *testing.T) {
...
+			break
+		}
+	}
+
 	body, err := ioutil.ReadAll(io.LimitReader(r.Body, r.ContentLength))
 	tests.Assert(t, err == nil)
 	r.Body.Close()
-	tests.Assert(t, strings.Contains(string(body), "Default Block Hosting Volume size is less than block volume requested."))
+	tests.Assert(t, strings.Contains(string(body), "Default Block Hosting Volume size is less than block volume requested."), "expected: Default Block Hosting Volume size is less than block volume requested", "got: %v", err)
+
```
Comment (obnoxxx) [inline apps/glusterfs/app_block_volume_test.go]: extra blank line

File: apps/glusterfs/app_block_volume_test.go [hunk 2]
```diff
@@ -178,11 +179,28 @@ func TestBlockVolumeLargerThanBlockHostingVolume(t *testing.T) {
...
-	tests.Assert(t, r.StatusCode == http.StatusBadRequest)
+	tests.Assert(t, r.StatusCode == http.StatusAccepted)
+	location, err := r.Location()
+	tests.Assert(t, err == nil)
+
+	for {
+		r, err = http.Get(location.String())
+		tests.Assert(t, err == nil)
+		if r.Header.Get("X-Pending") == "true" {
+			fmt.Fprintf(os.Stdout, "rtalurlogs")
```
Comment (obnoxxx) [inline apps/glusterfs/app_block_volume_test.go]: rtalurlogs?

**Comment by obnoxxx** [comment]
needs rebase

**Comment by raghavendra-talur** [comment]
closes #957

**Comment by raghavendra-talur** [comment]
@phlogistonjohn @obnoxxx this is ready for review.

**Pull request merged**